dockers/relaydc/add-instance

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl
```

Comment:
@drortirosh this means you're the only one going to work with this file

Reply:
It's a sample of how to add an entry to the dinner-compose file.
It duplicates the gsn1 entry, and rename it to the next number. 
It's not in use, and will never be used automatically. Only when we explicitly define a configuration with me server, which will required manual editing anyways.